check

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+from scripts.tool_runner import run_cmd
```

Comment:
I considered a few different names for the folder like `writer_tool_runner`. I wanted to emphasize this is only a runner of the tools. We should not define any tools in this folder.

I could imagine that maybe we'll have other private repositories like Functions install this new tool runner project as a VCS requirement, if it gets more complex. I'm not worried about that for now, but we should keep it in mind. Maybe, for example, we should move some `./start` code to this new folder.